packages/core/elasticsearch/core-elasticsearch-client-server-internal/src/client_config.test.ts

```diff
@@ -125,12 +125,44 @@ describe('parseClientOptions', () => {
           false,
           kibanaVersion
         );
-        expect(options.agent).toHaveProperty('maxSockets', 1024);
+        expect(options.agent).toHaveProperty('maxTotalSockets', 1024);
```

Comment:
Perhaps we could update the test description (L122) to reflect that `maxTotalSockets` is populated from `maxSockets`, WDYT?